openwisp_firmware_upgrader/api/views.py

```diff
@@ -0,0 +1,117 @@
+from rest_framework import generics
+from rest_framework.permissions import IsAuthenticated
+from swapper import load_model
+
+from openwisp_users.api.authentication import BearerAuthentication
+
+from .serializers import (
+    BatchUpgradeOperationSerializer,
+    BuildSerializer,
+    CategorySerializer,
+    FirmwareImageSerializer,
+)
+
+BatchUpgradeOperation = load_model('firmware_upgrader', 'BatchUpgradeOperation')
+Build = load_model('firmware_upgrader', 'Build')
+Category = load_model('firmware_upgrader', 'Category')
+FirmwareImage = load_model('firmware_upgrader', 'FirmwareImage')
+
+
+class ProtectedAPIMixin(object):
+    authentication_classes = [BearerAuthentication]
```

Comment:
can you add `SessionAuthentication`? It will make it easier for us to test the API via the browsable API interface.